samples/ipc_hardware_update/main.c

```diff
@@ -1,16 +1,36 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license.
-// See LICENSE file in the project root for full license information.
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
 
 #include "az_ulib_ipc_api.h"
+#include "az_ulib_registry_api.h"
 #include "az_ulib_result.h"
 #include "contoso_display_200401.h"
 #include "contoso_display_480401.h"
 #include "my_consumer.h"
 #include <inttypes.h>
+#include <stdint.h>
 #include <stdio.h>
 
 static az_ulib_ipc_control_block ipc_control_block;
+
+#define REGISTRY_PAGE_SIZE 0x800
+
+/* Static memory to store registry information. */
+static uint8_t registry_buffer[REGISTRY_PAGE_SIZE * 2];
```

Comment:
I know the buffers here are used because there is no FLASH for this sample, do you think we can put in some comments here to say that?

Reply:
#Pending